internal/infrastructure/database/database.go

```diff
@@ -0,0 +1,50 @@
+package database
+
+import (
+	"fmt"
+	"log/slog"
+	"time"
+
+	"gorm.io/gorm"
+)
+
+type Database struct {
+	Session *gorm.DB
+}
+
+// MustNewDatabase Establish database connection and migrate tables before
+// returning database.Database struct.
+func MustNewDatabase(d gorm.Dialector, gc gorm.Config, retryTimes int) Database {
```

Comment:
Curious about "Must".  Where did this come from (link)?

Reply:
Also, @jha-captech tells me it is a convention in the Go community.